modules/howtos/pages/encrypting-using-sdk.adoc

```diff
@@ -1,15 +1,175 @@
 = Field Level Encryption from the SDK
-:description: The Field Level Encryption library enables encryption and decryption of JSON fields, to support FIPS-140-2 compliance.
+:description: A practical guide for getting started with Field-Level Encryption, showing how to encrypt and decrypt JSON fields using the Python SDK.
 :page-topic-type: howto
 :page-edition: Enterprise Edition
-:page-status: Developer Preview
-// // :page-aliases: ROOT:encrypting-using-sdk.adoc
+//:page-aliases: ROOT:encrypting-using-sdk.adoc
 
 [abstract]
 {description}
 
+For a high-level overview of this feature, see xref:concept-docs:encryption.adoc[].
 
-Client-side implementation of Field Level Encryption is available in the previous version of the SDK.
-It will be enabled in a future release of the third generation SDK.
+[#package]
+== Packaging
 
-A _Developer Preview_ is available in the xref:3.0@java-sdk:howtos:encrypting-using-sdk.adoc[Java SDK].
+The Couchbase Python SDK works together with the https://github.com/couchbase/python-couchbase-encryption[Python Couchbase Encryption^] library to provide support for encryption and decryption of JSON fields.
+This library includes cryptographic algorithms you can use out of the box, and provides a framework for implementing your own crypto components.
+
+NOTE: The encryption code is packaged as an optional library because it is released under a https://github.com/couchbase/python-couchbase-encryption/blob/master/LICENSE[different license^] than the Couchbase Python SDK.

```

Comment:
We had a discussion with Matt about this, the note might need to change a little bit as we need to refer to the `Licence agreement` and `Enterprise Subscription License` it seems:

See https://github.com/couchbase/docs-sdk-go/pull/136/files as an example (though still pending review), @RichardSmedley is it ok for this to be updated as I've done in the linked PR?

Reply:
Thanks @maria-robobug - yes, please update.  :+1:

Reply:
Done.